platform_versioned_docs/version-23.4/getting-started/rnaseq.mdx

```diff
@@ -0,0 +1,749 @@
+      print(p)
+      # Save to file (change `.png` to `.pdf` to create PDF files)
+      ggsave(
+        paste0("volcano_plot_", name, ".png"),
+        p,
+        width = 8,
+        height = 6,
+        dpi = 300
+      )
+    }
+    ```
+
+1. Create a heatmap of the top 50 differentially expressed genes:
+
+    :::info
+    Heatmaps in RNA-seq analysis provide a color-coded representation of gene expression levels across multiple samples or conditions, enabling the visualization of expression patterns and sample clustering based on similarity.
+    :::
+
+    ```r
+    # Create a heatmap of top 50 differentially expressed genes
+    # Get top 50 DE genes from each comparison
+    top_genes <- unique(unlist(lapply(results, function(x)
+      rownames(x$table)[1:50])))
+
+    # Get log-CPM values for these genes
+    log_cpm <- cpm(y, log = TRUE)
+    top_gene_expr <- log_cpm[top_genes, ]
+
+    # Print dimensions of top_gene_expr
+    print(dim(top_gene_expr))
+
+    # Create a color palette
+    my_palette <- colorRampPalette(c("blue", "white", "red"))(100)
+
+    # Create a heatmap using heatmap.2
+    # Display in RStudio
+    heatmap.2(
+      as.matrix(top_gene_expr),
+      scale = "row",
+      col = my_palette,
+      trace = "none",
+      dendrogram = "column",
+      margins = c(5, 10),
+      labRow = FALSE,
+      ColSideColors = rainbow(length(unique(y$samples$group)))[factor(y$samples$group)],
+      main = "Top DE Genes Across Samples"
+    )
+
+    # Save heatmap to file (change `png` to `pdf` to create a PDF file)
+    png("heatmap_top_DE_genes.png",
+        width = 1000,
+        height = 1200)
+    heatmap.2(
+      as.matrix(top_gene_expr),
+      scale = "row",
+      col = my_palette,
+      trace = "none",
+      dendrogram = "column",
+      margins = c(5, 10),
+      labRow = FALSE,
+      ColSideColors = rainbow(length(unique(y$samples$group)))[factor(y$samples$group)],
+      main = "Top DE Genes Across Samples"
+    )
+    dev.off()
+
+    # Print the number of top genes in the heatmap
+    print(paste("Number of top DE genes in heatmap:", length(top_genes)))
+    ```
+
+![RStudio plots](./_images/rstudio.gif)
+
+### Collaborate in the data studio
+
+To share your results or allow colleagues to perform exploratory analysis, share a link to the data studio by selecting the options menu for the data studio you want to share, then select **Copy data studio URL**. With this link, other authenticated users with the **Connect** [role](../orgs-and-teams/roles.mdx) (or greater) can access the session directly.
+
+## RNA-Seq data and requirements
+
+RNA-Seq data typically consists of raw sequencing reads from high-throughput sequencing technologies. These reads are used to quantify gene expression levels and discover novel transcripts. A typical RNA-Seq dataset can range from a few GB to several hundred GB, depending on the number of samples and the sequencing depth.
+
+### nf-core/rnaseq performance in Platform
```

Comment:
Not a blocker, but approaching this from a scientist's point of view, I'm left wondering how many times this was run to generate these values. Simply, was this a single run? Or was the analysis run multiple times to generate these numbers?

Reply:
Added small clarification that the benchmark is a single run of the pipeline on each CE.